### PR TITLE
Added More Perks for DLCs and related cleaning data

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -1040,8 +1040,29 @@ plugins:
...
+    url: [ 'https://www.nexusmods.com/newvegas/mods/35979' ]
     tag:
       - Names
+  - name: 'More Perks for Dead Money.esm'
+    dirty:
+      - <<: *quickClean
+        crc: 0x129204A2
+        util: '[FNVEdit v4.0.1i EXPERIMENTAL](https://www.nexusmods.com/newvegas/mods/34703)'
+        itm: 301
+    clean:
```
Comment (Ortham) [inline masterlist.yaml:1052]: A plugin can't be clean and also have wild edits... The wild edit list items should also come under `dirty:`.

**Review by Ortham — CHANGES_REQUESTED** [review]

File: masterlist.yaml (hunk 1 above)
Comment (ghost) [reply masterlist.yaml:1052]: 👍 I'll update that

Comment (ghost) [reply masterlist.yaml:1052]: Should be fixed. If there are any other complaints please let me know.,

Comment (Ortham) [reply masterlist.yaml:1052]: I don't see any changes, did you commit them to the right branch / push the branch?

Comment (ghost) [reply masterlist.yaml:1052]: https://github.com/gutfuckllc/falloutnv/commit/21bc1c898d56b9645dd44f573c41ddbcb901b7aa

It appears on my end. Not sure what's wrong here. Git is such an headache sometimes

Comment (ghost) [reply masterlist.yaml:1052]: Should I close and reopen a pull request?

**Pull request closed without merging**